src/azure-cli/azure/cli/command_modules/synapse/manual/operations/accesscontrol.py

```diff
@@ -141,20 +141,18 @@ def _resolve_object_id(cmd, assignee, fallback_to_object_id=False):
                            "Please using --assignee-object-id GUID to specify assignee accurately"
                            .format(assignee=assignee))
 
-        return result[0].object_id
-    except (CloudError, GraphErrorException):
+        return result[0]["id"]
+    except (CloudError, GraphError):
```

Comment:
`CloudError` will never be raised by `GraphClient`. Capturing `GraphError` is enough.